src/components/Todo.js

```diff
@@ -1,24 +1,33 @@
 import React, { Component } from 'react';
 
 class Todo extends Component {
-  render() {
-    return (
-      <li>
+	constructor(props) {
+		super(props);
+	}
+	
+	render() {
+		return (
+			<li>
 				<div className="view">
 					<input
 						className="toggle"
 						type="checkbox"
-						checked={false}
-						onChange={() => console.log('Toggle Todo')}
+						checked={this.props.done}
+						onChange={(e) => this.props.onChange(e.target.checked)}
 					/>
+					
 					<label>
 						{this.props.label}
 					</label>
-					<button className="destroy" onClick={() => console.log('Destroy Todo')} />
+					
+					<button 
+						className="destroy"
+						onClick={() => this.props.onRemove()}
```

Comment:
Perfectly fine.  Could also be written as:
`onClick={this.props.onRemove}` (if bound beforehand)
OR
`onClick={this.props.onRemove.bind(this)}` (if you want to bind it inline)